test/e2e/apimachinery/namespace.go

```diff
@@ -246,4 +248,29 @@ var _ = SIGDescribe("Namespaces [Serial]", func() {
 	ginkgo.It("should always delete fast (ALL of 100 namespaces in 150 seconds) [Feature:ComprehensiveNamespaceDraining]",
 		func() { extinguish(f, 100, 0, 150) })
 
+	ginkgo.It("should patch a Namespace", func() {
+		ginkgo.By("creating a Namespace")
+		namespaceName := "nspatchtest"
+		_, err := f.ClientSet.CoreV1().Namespaces().Create(&v1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: namespaceName,
+			},
+		})
+		framework.ExpectNoError(err, "failed creating Namespace")
+
+		ginkgo.By("patching the Namespace")
+		nspatch := fmt.Sprintf(`{"metadata":{"labels":{"testProperty":"testValue"}}}`, namespaceName)
+		_, err = f.ClientSet.CoreV1().Namespaces().Patch(namespaceName, types.StrategicMergePatchType, []byte(nspatch))
+		framework.ExpectNoError(err, "failed to patch Namespace")
+
+		ginkgo.By("get the namespace and ensuring it has a LabelSelector")
```

Comment:
"a LabelSelector" should be just "the label".

Reply:
Updated in https://github.com/kubernetes/kubernetes/pull/86854/commits/8abdc97a53d768394613cd89e467d4ac8b8d8c9a